scripts/zkai_attention_kv_fused_softmax_table_microprofile_gate.py

```
@@ -0,0 +1,937 @@
+    for version in aggregate["proof_backend_versions"]:
+        require_str(version, "proof_backend_version")
+    if tuple(aggregate["exposed_relation_width_profiles"]) != EXPECTED_EXPOSED_RELATION_WIDTH_PROFILES:
+        raise FusedSoftmaxTableMicroprofileGateError("exposed relation-width profile drift")
+    section_totals = aggregate["section_totals"]
+    if not isinstance(section_totals, dict) or set(section_totals) != set(PROOF_SECTION_KEYS):
+        raise FusedSoftmaxTableMicroprofileGateError("section total field drift")
+    if sum(section_totals.values()) != aggregate["total_section_payload_bytes"]:
+        raise FusedSoftmaxTableMicroprofileGateError("section total sum drift")
+    buckets = aggregate["bucket_totals"]
+    if not isinstance(buckets, dict):
+        raise FusedSoftmaxTableMicroprofileGateError("bucket totals must be object")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)